h2o-algos/src/main/java/hex/StackedEnsembleModel.java

```diff
@@ -204,14 +204,16 @@ public void doScoreOrCopyMetrics(Job job) {
     this._output._training_metrics = doScoreMetricsOneFrame(this._parms.train(), job);
     // Validation metrics can be copied from metalearner (may be null).
     // Validation frame was already piped through so there's no need to re-do that to get the same results.
+    //this._output._validation_metrics = this._output._metalearner._output._validation_metrics.deepCloneWithDifferentModelAndFrame(this, this._output._metalearner._parms.valid());  #valid or train?
```

Comment:
Is this comment needed?

Reply:
It's a reminder for the future that we might want to look into doing a deep copy for validation metrics as well (not needed at this time).